### ISSUE TITLE
Have a way to provide options to override jest test config

### ISSUE THREAD

**Issue by ardok** [fusionjs/fusion-cli#275]
At the moment, there's no way for devs to override the jest config

### Type of issue

Feature request

### Description

One simple use case is to specify a setup files. Say I want to set `setupFiles` for node tests and a different setup file for browser tests.

### Current behavior

There's no clear entry point for the tests nor setup file.

### Expected behavior

Maybe have an option to pass in a config file so that devs could override the config with their own config file.
Ideally, different for browser and node.
For example, in my browser tests, I do this:
```
{
  ...
  "setupFiles": [
    "<rootDir>/src/test/browser/setup.js"
  ]
}
```

And in node tests:
```
{
  ...
  "setupFiles": [
    "<rootDir>/src/test/node/setup.js"
  ]
}
```

### Steps to reproduce

N/A

### Your environment

* fusion-cli version: 1.1.0

* Node.js version (`node --version`): 8.9.4

* npm version (`npm --version`): 5.6.0

* Operating System: Mac OS

**Comment by KevinGrandon** [comment]
@ardok Thanks for filing. What specific things are you looking to perform inside of setupFiles? If there's some quality of life changes that we can do for all apps we would definitely like to do that first.

We can look at either providing a config option, or having the CLI look for a file at a specific path to use.

**Comment by ardok** [comment]
Hmm, simplest example would be:

For browser env:
I want to sent Enzyme adapter and mock moment module (or Date).

For node env:
I just want to mock moment module (or Date).

**Comment by ardok** [comment]
Also, another example is for `collectCoverageFrom` options in jest.
I have this:
```
  "collectCoverageFrom": [
    "!**/node_modules/**",
    "<rootDir>/src/client/**/*.js",
    "<rootDir>/src/server/**/*.js",
    "<rootDir>/src/shared/**/*.js",
    "!<rootDir>/src/shared/types/**/*.js"
  ],
```

You see that I have a custom directory there `"!<rootDir>/src/shared/types/**/*.js"`.
It just contains js file with lots of `export type` for flow.
You can imagine that some other projects might have different directory structure.
So, it'd be great to be able to also customize those.

**Comment by mlmorg** [comment]
We will fix #274 instead of this, as things should just work out of the box.
